packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkDecoding.ts

```diff
@@ -38,15 +38,25 @@ import {
 	type EncodedValueShape,
 	SpecialField,
 } from "./format.js";
-import type { IIdCompressor, SessionSpaceCompressedId } from "@fluidframework/id-compressor";
+import type { IIdCompressor, OpSpaceCompressedId, SessionId } from "@fluidframework/id-compressor";
 
+export interface IdDecodingContext {
+	idCompressor: IIdCompressor;
+	/**
+	 * The creator of any local Ids to be decoded.
+	 */
+	originatorId: SessionId;
+}
 /**
  * Decode `chunk` into a TreeChunk.
  */
-export function decode(chunk: EncodedFieldBatch, idCompressor: IIdCompressor): TreeChunk[] {
+export function decode(
+	chunk: EncodedFieldBatch,
+	idCompressorContext: { idCompressor: IIdCompressor; originatorId: SessionId },
+): TreeChunk[] {
 	return genericDecode(
 		decoderLibrary,
-		new DecoderContext(chunk.identifiers, chunk.shapes, idCompressor),
+		new DecoderContext(chunk.identifiers, chunk.shapes, idCompressorContext),
```

Comment:
I'd also do a find-replace for idCompressorContext -> idDecodingContext :)

Reply:
renamed variable to `idDecodingContext`